README.md

```diff
@@ -1,24 +1,40 @@
-#TODO: Create title with heading
+#Scrape for books
 
-#TODO: Create Short description
-Bienvenue dans le code qui va scrapper vos données.
-Vous avez envi de recuperer des données de une ou plusieurs pages ? Pas de problèmes avec ce script, vous le pouvez maintenant.
+##Description
+You want takes data from website " _**bookstoscrape**_ ", this script can make that for you.
```

Comment:
This script has been created to scrape books details from the website https://books.toscrape.com.